### PR TITLE
BSM-144: Update the header of the Ops Board to use the same styling as the rest of the application

### PR DESCRIPTION
**Pull request by christianpape**
Bamboo: http://bamboo.internal.opennms.com:8085/browse/OPENNMS-ONMS479
Jira: http://issues.opennms.org/browse/BSM-144

I did an implementation without using the OnmsHeaderProvider because 
the Ops Board/Panel has its own input fields and make no use of the original 
OpenNMS menu bar.

### CODE REVIEW

File: features/vaadin-dashboard/src/main/java/org/opennms/features/vaadin/dashboard/ui/HeaderLayout.java [hunk 1]
```diff
@@ -59,17 +65,19 @@ public HeaderLayout() {
...
         setSpacing(true);
         setWidth("100%");
+        setHeight(64, Unit.PIXELS);
 
         /**
          * Adding the logo
          */
         Link link = new Link(null, new ExternalResource("/opennms/index.jsp"));
-        link.setIcon(new ThemeResource("img/logo.png"));
+        link.setIcon(new ExternalResource("/opennms/images/horizon_logo.svg"));
```
Comment (mvrueden) [inline features/vaadin-dashboard/src/main/java/org/opennms/features/vaadin/dashboard/ui/HeaderLayout.java]: Can you set the icon based on css? 
This is definetly causing trouble for a Meridian release.

Comment (christianpape) [inline features/vaadin-dashboard/src/main/java/org/opennms/features/vaadin/dashboard/ui/HeaderLayout.java]: It's defined in a ftl-template and not in a scss/css-File. This is a general problem and cannot be addressed in this issue.

**Comment by j-white** [comment]
The problem with this approach is that it makes it difficult to maintain alternate branding (i.e. Meridian).

Can't we just make the necessary changes to the OnmsHeaderProvider API?

**Comment by christianpape** [comment]
The logo is now configurable in the dashboard's theme CSS-file. Is this okay? I've really no clue how to insert Vaadin-based components in the header's HTML-fragment provided by the OnmsHeaderProvider.

**Comment by j-white** [comment]
Yeah, that's better. Can you re-target this patch for foundation-2016?

**Comment by j-white** [comment]
I'll cherry pick this to foundation-2016 once it's merged.

**Pull request merged**